README.md

```diff
@@ -4,16 +4,14 @@
 
 ## Introduction
 
-This repository contains a userscript designed to remove the annoying "Ad blocker is not allowed on Youtube" popup. Follow the instructions below to get started. We also have an undetected way of skipping ads if you get blocked from the video player.
-
+This repository contains a [Tampermonkey](https://github.com/Tampermonkey/tampermonkey) userscript designed to remove the annoying "Ad blocker is not allowed on Youtube" popup. Follow the instructions below to get started.
```

Comment:
It's not specific to TamperMonker, I guess there is other userscript extension that exists for other browsers.

Reply:
I see, thank you!